AdvancedStateAndSideEffectsCodelab/app/src/main/java/androidx/compose/samples/crane/base/ExploreSection.kt

```diff
@@ -114,11 +118,10 @@ private fun ExploreItem(
     ) {
         ExploreImageContainer {
             Box {
-                val painter = rememberImagePainter(
-                    data = item.imageUrl,
-                    builder = {
+                val painter = rememberAsyncImagePainter(
+                    Builder(LocalContext.current).data(data = item.imageUrl).apply(block = fun Builder.() {
                         crossfade(true)
-                    }
+                    }).build()
```

Comment:
Have you checked [#801 ](https://github.com/android/compose-samples/pull/801/files) in compose samples? 
```
        val painter = rememberAsyncImagePainter(
            model = ImageRequest.Builder(LocalContext.current)
                .data(item.imageUrl)
                .crossfade(true)
                .build()
        )
```